### PR TITLE
Fix the `combine` option for overlay modes inside blankLines

### PR DESCRIPTION
**Pull request by tberman**
Before, the base methods were being called but nothing was being returned in either case. This should fix it both for normal and combine overlays.

### CODE REVIEW

File: addon/mode/overlay.js [hunk 1]
```diff
@@ -76,8 +76,15 @@ CodeMirror.overlayMode = function(base, overlay, combine) {
...
 
     blankLine: function(state) {
-      if (base.blankLine) base.blankLine(state.base);
-      if (overlay.blankLine) overlay.blankLine(state.overlay);
+      var baseToken, overlayToken;
+      if (base.blankLine) baseToken = base.blankLine(state.base);
+      if (overlay.blankLine) overlayToken = overlay.blankLine(state.overlay);
+
+      if (combine) {
+        return baseToken + " " + overlayToken;
```
Comment (marijnh) [inline addon/mode/overlay.js]: This looks like it will often append undefined/null to a string.

Comment (tberman) [inline addon/mode/overlay.js]: Good catch. Will update when I get back to my computer.

**Comment by tberman** [comment]
Alright, @marijnh, should be fixed.

**Comment by marijnh** [comment]
Thanks. Merged as d0cde7f8470d66

**Pull request closed without merging**